src/components/tables/UsersDataTable.vue

```diff
@@ -66,7 +66,7 @@ export default {
       this.users.forEach(user => {
         this.rows.push(
           [
-            { value: user.email },
+            { value: { title: user.email, userId: 37875}, type: 'action' },
```

Comment:
what's this user id?

Reply:
@squallstar That is my user ID taken from the Fliplet API threw the `Fliplet.User.fetch()` method.
Used for the same reasons as above.